src/polyfill.js

```diff
@@ -0,0 +1,7 @@
+let global = Function('return this')();
+import implementation from './implementation';
+
+export default function() {
+  let p = global.Protocol;
```

Comment:
however, doing just `Protocol` might be more robust than pulling it off `global`?